### ISSUE TITLE
Luminosity ratio is not  sufficient for close button on MessageBar. It should be greater or equal to 4.5 : 1

### ISSUE THREAD

**Issue by v-ajrai** [microsoft/fluentui#15016]
Luminosity ratio is not  sufficient for close button on MessageBar on Office Ui Fabric . It should be greater or equal to 4.5 : 1

### Environment Information
OS: Win 10
Browser:  Chrome

### Describe the issue:
Luminosity ratio is not  sufficient for close button  on MessageBar on Office Ui Fabric . It should be greater or equal to 4.5 : 1

#### Providing  reproduction of the issue
https://developer.microsoft.com/en-us/fluentui#/controls/web/messagebar
![external_2009663](https://user-images.githubusercontent.com/62233982/93069063-d5c3d900-f69a-11ea-8188-a2175ab60162.png)

#### Actual behavior:
Luminosity ratio is not  sufficient for close button on MessageBar.

#### Expected behavior:
Luminosity ratio should be sufficient for close button on MessageBar.

**Comment by miroslavstastny** [comment]
Assigning to @ecraig12345 as MessageBar owner. Probably needs design input.

**Comment by ecraig12345** [comment]
It looks like this was fixed at some point after the issue was filed. With the current colors, [the contrast ratio is 10.99:1](https://contrast-ratio.com/#rgb%2850%2C%2049%2C%2048%29-on-rgb%28253%2C%20231%2C%20233%29). So I'm going to close the issue.
